.gitignore

```diff
@@ -33,3 +33,4 @@ coverage.xml
 
 # Rever
 rever/
+*.swp
```

Comment:
Putting it here implies that it's related to Rever, which isn't really true. I'd suggest putting it under `*~`, which I think is used in a similar way.

Reply:
Done.